src/components/forms/fragments/overview/OverviewStorageMethod.tsx

```diff
@@ -0,0 +1,47 @@
+import React, { SFC } from 'react';
+import { FormattedMessage } from 'react-intl';
+import { Col, Row } from 'reactstrap';
+import { WinterStorageMethod } from '../../../../__generated__/globalTypes';
+
+const OverviewStorageMethod: SFC<{
+  storageMethod: WinterStorageMethod | undefined;
+  registrationNumber: string | undefined;
+}> = ({ storageMethod, registrationNumber }) => {
+  const storageLabel =
+    storageMethod === WinterStorageMethod.ON_TRESTLES
+      ? 'form.winter_storage_method.field.storage_method.on_trestles'
+      : 'form.winter_storage_method.field.storage_method.on_trailer';
```

Comment:
so, if it's `undefined` it will show `ON_TRAILER`? don't think `undefined` should be allowed at all, but if it's needed be the react logic, we should probably check that the storage method is defined first

Reply:
If its undefined this component will not render at all. `{isWinterStorageFormMode && values.storageMethod && `

Reply:
ah, right, missed that